src/main/kotlin/hwr/oop/projects/peakpoker/core/player/Player.kt

```diff
@@ -46,7 +59,9 @@ class Player(
   }
 
   fun allIn() {
-    setBetAmount(this.chips)
+    val totalBet = chips + bet
+    chips = 0
+    setBetAmount(totalBet)
```

Comment:
@juliankuecken why did yiu changed the logic here?
```suggestion
    setBetAmount(this.chips)
```

Reply:
I think that previously there was an issue with the all-in function. This was pointed out to me by the test in `GameTestCustomPlayer.kt` on lines _229 to 245_.
Here i would kindly ask you to play around with the tests as well as the previous implementations and make up your own mind. (Feedback of course to me please).! 🙌

Reply:
Are you sure that you have the scenario of a player betting before (either blind or by himself) and then going allIn afterwards? Are the tests correct?
If that is so then please find the issue, why the perfectly reasonable test i implemented in the mentioned lines fails.

Reply:
@dev-laky I slept another night over the changes, tested further and came to the conclusion that this change is correct.
Please somewhat hurry with your review! ;)